bofire/surrogates/robust_single_task_gp.py

```diff
@@ -0,0 +1,146 @@
+from typing import Dict, Optional
+
+import pandas as pd
+import torch
+from botorch.fit import fit_gpytorch_mll
+
+# from bofire.data_models.molfeatures.api import MolFeatures
+from botorch.models.relevance_pursuit import (
+    backward_relevance_pursuit,
+    forward_relevance_pursuit,  # noqa: F401
+)
+from botorch.models.robust_relevance_pursuit_model import (
+    RobustRelevancePursuitSingleTaskGP,
+)
+from botorch.models.transforms.outcome import Standardize
+from gpytorch.mlls import ExactMarginalLogLikelihood
+
+import bofire.kernels.api as kernels
+import bofire.priors.api as priors
+from bofire.data_models.enum import OutputFilteringEnum
+
+# from bofire.data_models.surrogates.api import SingleTaskGPSurrogate as DataModel
+from bofire.data_models.surrogates.api import RobustSingleTaskGPSurrogate as DataModel
+from bofire.data_models.surrogates.scaler import ScalerEnum
+from bofire.surrogates.botorch import BotorchSurrogate
+from bofire.surrogates.single_task_gp import SingleTaskGPSurrogate
+from bofire.surrogates.trainable import TrainableSurrogate
+from bofire.surrogates.utils import get_scaler
+from bofire.utils.torch_tools import tkwargs
+
+
+class RobustSingleTaskGPSurrogate(
+    SingleTaskGPSurrogate, BotorchSurrogate, TrainableSurrogate
+):
+    """
+    Robust Relevance Pursuit Single Task Gaussian Process Surrogate.
+
+    A robust single-task GP that learns a data-point specific noise level and is therefore more robust to outliers.
+    See: https://botorch.org/docs/tutorials/relevance_pursuit_robust_regression/
+    Paper: https://arxiv.org/pdf/2410.24222
+
+    Attributes:
```

Comment:
Add the docstring also to the data model. There people interact most of the time with it. And there they can actually set the parameters, here they can only set the data model on init, which is a bit misleading.